### PR TITLE
Update travel and other pages to add web services language.

### PR DESCRIPTION
**Pull request by RoAnnC1**
Multiple minor updates to spacing and heading casing in travel docs.

### CODE REVIEW

File: src/api-reference/expense/allocations/expense-allocations.markdown [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Allocations
 layout: reference
 ---
 
-This resource can be used to retrieve information about the allocations that are associated with an entry in an expense report. v3.0
+The SAP Concur Allocations API allows for the retrieval of allocation information as it relates to a Report ID, Entry ID, or Itemization ID. Using this API allows for an in-depth review of Expense Report Data and how that data has been allocated in SAP Concur. The Allocations Web Service allows for the programmatic gathering of details on how the Expense Report data was allocated by the report owner, making it ideal for Data Gathering, Expense Reporting, and Validations. This resource can be used to retrieve information about the allocations that are associated with an entry in an expense report. v3.0
```
Comment (sarraloew) [inline src/api-reference/expense/allocations/expense-allocations.markdown]: Remove the last sentence and the version. "Expense Report Data" shouldn't be capitalized. Change "Allocations Web Service" to "Allocations API"

File: src/api-reference/expense/allocations/expense-allocations.markdown [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Allocations
...
 
 
 * [Retrieve All Allocations Per Entry or Report](#get)
```
Comment (sarraloew) [inline src/api-reference/expense/allocations/expense-allocations.markdown:8]: On line 36, the request example includes a parameter that we don't support, please remove the "&user=ALL" part of the request.

File: src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown [hunk 3]
```diff
@@ -3,12 +3,7 @@ title: Itinerary Web Service (TMC/Third-Party)
...
-
+The Concur Itinerary Web Service allows Travel Management Companies (TMC), SAP Concur clients, and third-party developers to view and create travel related events in the Concur Travel system. TMCs can post bookings for any travel type. This web service can also be used by SAP Concur clients and third-party developers to request trip information for Concur users. The public Itinerary XSD can be found [here.](/api-reference/travel/itinerary/ItinServices_Public_0.xsd)  In addition, the GetList XSD can be found [here.](/api-reference/travel/itinerary/GetListOfItinerariesImport_v1_1.xsd)
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown]: SAP Concur

File: src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown [hunk 4]
```diff
@@ -1801,9 +1757,8 @@ Authorization: OAuth {access token}
 </Car>
 ```
 
-# <a name="objects"></a>Booking Object Elements
+## <a name="objects"></a>Booking Object Elements
 
-## Booking Object Elements - TMCs and Third-party Developers
 The booking elements contain many child elements. For ease of use, these elements are divided into the Core Elements, which are the most frequently used, and Additional Elements, which are not often used but are supported by the Itinerary web service. Some elements only appear if the travel supplier created the booking. Elements are marked as required if they must be supplied for a new booking.  
 
 **NOTE:** TripLink - Open Booking suppliers see a targeted subset of these fields. Refer to the documentation here for the TripLink - Open Booking supplier booking object elements.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown]: Looks like a link is missing here- "here" should point to this page: https://developer.concur.com/api-reference/travel/itinerary/trip/trip-resource.html

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 5]
```diff
@@ -395,7 +395,7 @@ The response returns subset of the elements described in the following tables de
 
 | Element Name | Data Type |Description
 | --------- | --------- | -------
-|Segments	|array	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
+|Segments	|array	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, SAP Concur client,  third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs, clients, and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Final word should be plural.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 6]
```diff
@@ -693,7 +693,7 @@ Where `access_token` is the OAuth 2.0 access token of the user whose trip you wa
...
-| Segments | optional	 | Y | array	| List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. For TripLink suppliers, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
+| Segments | optional	 | Y | array	| List of segments in this booking. The child elements included in this element vary depending on whether a TMC, client, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs, clients, and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. For TripLink suppliers, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: same change here.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 7]
```diff
@@ -712,7 +712,7 @@ Where `access_token` is the OAuth 2.0 access token of the user whose trip you wa
...
-| Segments | optional | Y | array | List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
+| Segments | optional | Y | array | List of segments in this booking. The child elements included in this element vary depending on whether a TMC, client, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs, clients, and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: And same here.

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/api-reference/expense/allocations/expense-allocations.markdown (hunk 2 above)
Comment (RoAnnC1) [reply src/api-reference/expense/allocations/expense-allocations.markdown:8]: Done

File: src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown [hunk 8]
```diff
@@ -279,7 +279,8 @@ Name |  Description
 `BaseFareNucCurrency` |  The [3-letter ISO 4217 currency code][5] for the base fare in NUC. |
 `DateCreatedUtc` |  The date the quote was created, in UTC. Format: YYYY-MM-DDThh:mm:ss |
 `DateModifiedUtc` |  The date the quote was last modified, in UTC. Format: YYYY-MM-DDThh:mm:ss |
-`Endorsements` |  Notes from the airline if it endorses the ticket as acceptable on a different airline. |
+`Endorsements` |
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary-tmc-thirdparty/index.markdown]: Something went weird here.

**Review by sarraloew — CHANGES_REQUESTED** [review]

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Comment by charlieconcur** [comment]
:shipit:

**Pull request merged**